tests/tracing/test_dnatracing_methods.py

```diff
@@ -19,17 +19,16 @@
 MIN_SKELETON_SIZE = 10
 
 
-@pytest.fixture
+@pytest.fixture()
 def dnatrace() -> dnaTrace:
-    """Instantiated object of class dnaTrace for use in tests."""
-    _dnatrace = dnaTrace(
+    """DnaTrace object for use in tests."""
```

Comment:
Should read dnaTrace rather than DnaTrace here to align with code

Reply:
`noqa: D403` added, ta :+1: